repair_scheduled_date_calendar_view/readme/CONTEXT.md

```
@@ -0,0 +1 @@
+Sometimes it can be really helpful to have an overview of the planned repair orders. This can be especially useful when you need to allocate a free slot to schedule a new repair order.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)